tests/compile_cmake.sh

```diff
@@ -42,23 +32,22 @@ else
   clean_before=${5:-YES}
   clean_after=${6:-YES}
 fi
-BUILD_DIR=build_${BUILD_NAME}
+
+BUILD_DIR=$(pwd)/build_${BUILD_NAME}
 
 # ----------------------------------------------------------------------
 # Make sure we have reasonable number of threads.
 
 if [[ $MACHINE_ID == cheyenne.* ]] ; then
     MAKE_THREADS=${MAKE_THREADS:-3}
 elif [[ $MACHINE_ID == wcoss_dell_p3 ]] ; then
-    MAKE_THREADS=${MAKE_THREADS:-4}
+    MAKE_THREADS=${MAKE_THREADS:-1}
```

Comment:
May I ask why the MAKE_THREADS is changed to one? Is it the reason the model compile is taking longer time now?

Reply:
This change is from @DusanJovic-NOAA .

Reply:
Yes. Compilation now runs on compute nodes and on WCOSS Dell is using 'dev_transfer' queue which allows only one task. So each COMPILE step is running little slower but on the other hand we can simultaneously submit up to 10 COMPILE steps.